org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/Util.java

```diff
@@ -302,6 +302,22 @@ public static String readAll(String filename) throws FileNotFoundException, IOEx
         return convertLineBreak(ans);
     }
 
+    /** Returns the modified date of the given file
```

Comment:
Very small nitpick: can you please add a new line after `/**`, just to be consistent with the rest of the codebase?